build/shared-runtime.version

```diff
@@ -1 +1 @@
-1.2.0-beta-001304-00
\ No newline at end of file
+1.1.1;release/1.1.0,2.0.0-beta-001747-00;master
```

Comment:
I am a bit unclear about this mismatching NETCoreApp versions. I ran into this issue when doing `dotnet test` when it warned about the `2.0.0-beta-001747-00` version not available. This means that the test runner tool depends on this version and hence the error, is it?? If the mirror were to verify what would we be trying here? that the tools depend on the CLI's netcoreapp versions only?